### ISSUE TITLE
`O)` (Object) and `P)` (Position) RegEx options no longer required

### ISSUE THREAD

**Issue by fade2gray** [mmikeww/AHK-v2-script-converter#129]
Thank you for this script, it has been an immense help.

Working v1 example:
`RegExMatch(test_string, "O)(<RequiredPackages Count=""(\d+)"">)", match)`

Incorrect v2 converter output:
````RegExMatch(test_string, "O)(<RequiredPackages Count=`"(\d+)`">)", match)````

Acceptable v2 converter output:
````RegExMatch(test_string, "(<RequiredPackages Count=`"(\d+)`">)", match)````

**Comment by Banaanae** [comment]
O) and P) do get removed, It's looks like it has something to do with escaping the quotes? Something causes "match" to not be param 3 in the array, which prevents the remove O and P code from being ran

**Comment by fade2gray** [comment]
> O) and P) do get removed

I'm not sure that I understand you.

Working v1 code:
```ahk
test_string := "<RequiredPackages Count=""123"">"

RegExMatch(test_string, "O)(<RequiredPackages Count=""(\d+)"">)", match)

MsgBox, % "match.1 = " match.1 "`nmatch.2 = " match.2
```

Incorrect v2 converter output:
```ahk
test_string := "<RequiredPackages Count=""123"">" ; quotes have not been escaped with backticks

RegExMatch(test_string, "O)(<RequiredPackages Count=`"(\d+)`">)", match) ; quotes have been escaped correctly, but the O) option has not been removed, and the match variable has not been prepended with &.

MsgBox("match.1 = " match.1 "`nmatch.2 = " match.2)
```

Expected v2 converter output:
```ahk
test_string := "<RequiredPackages Count=`"123`">"

RegExMatch(test_string, "(<RequiredPackages Count=`"(\d+)`">)", &match)

MsgBox("match.1 = " match.1 "`nmatch.2 = " match.2)
ExitApp
```

**Comment by Banaanae** [comment]
![image](https://github.com/mmikeww/AHK-v2-script-converter/assets/83927639/f011cd2e-faf5-43ff-bd8f-7f6af42416bc)
Something's wrong in the backend that makes certain regex strings not remove O)

**Comment by fade2gray** [comment]
Ah, I understand now - thanks.

**Comment by Banaanae** [comment]
Fixed in #208
